app/components/pages/CustomFieldsSettings/tests/CreateEditCustomFieldContent.test.js

```
@@ -0,0 +1,187 @@
+      setEnumValueError: setEnumValueErrorMockFn,
+    },
+    customFieldMeta: {},
+    orgTimezone: {},
+    ...props,
+  };
+
+  return render(<Component {...propsObj} />);
+};
+
+describe('CreateEditCustomFieldContent', () => {
+  let errorMessage = 'Test error message';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'errorMessage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)